virt-v2v/cold/entrypoint.go

```diff
@@ -62,6 +63,12 @@ func main() {
 		fmt.Println("Error getting XML file:", err)
 		os.Exit(1)
 	}
+	// If needed, customize the VM
```

Comment:
since only non-boot disks are set as offline, there's no point in going there in case the VM has a single disk, which is supposed to be a very common case...